app/src/app/store/mapRenderSubs.ts

```diff
@@ -303,6 +311,18 @@ export const getRenderSubscriptions = (useMapStore: typeof _useMapStore) => {
       } 
     }
   );
+  
+  const filterCountiesSub = useMapStore.subscribe<[string|undefined, MapStore['getMapRef']]>(state => [state.mapOptions.currentStateFp, state.getMapRef],
+    ([stateFp, getMapRef]) => {
+      const mapRef = getMapRef()
+      if (!mapRef) return
+      const filterExpression = (stateFp ? ["==", "STATEFP", stateFp] : true) as any
+      COUNTY_LAYERS.forEach(layer => {
+        mapRef.setFilter(layer,  ["any", filterExpression])
+      })
```

Comment:
H: Need to check that the county layer is added first

<img width="1333" alt="image" src="https://github.com/user-attachments/assets/4125dff6-8db4-48c8-ab99-b86b46d0574e">

Reply:
Ah yep